commit ce326f093172ffd8548d760e1d46ed0a6201854b
Author: sveinung <sveinung@a0f10bec-cc02-0410-94fc-a9cfff90b4cd>
Date:   Tue Oct 27 15:44:23 2015 +0000

    Europe 1901: fix closest diplstate
    
    Change closest diplstate from "Never met" to the current diplstate for
    nations that have Peace or an Alliance. (It had been stored as "Never
    met" because of bug #23973)
    
    This is just a technical correction. No historical research involved.
    
    See bug #23978
    
    git-svn-id: svn://svn.gna.org/svn/freeciv/trunk@30232 a0f10bec-cc02-0410-94fc-a9cfff90b4cd

diff --git a/data/scenarios/europe_1901.sav b/data/scenarios/europe_1901.sav
index 92a1f25..d5cf28b 100644
--- a/data/scenarios/europe_1901.sav
+++ b/data/scenarios/europe_1901.sav
@@ -3401,7 +3401,7 @@ diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_c
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -3410,7 +3410,7 @@ diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_c
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -5168,8 +5168,8 @@ diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_c
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 5,5,0,0,0,0,FALSE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -6864,9 +6864,9 @@ ai.control=FALSE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
 5,5,0,0,0,0,FALSE,FALSE
-4,5,0,16,0,20,TRUE,FALSE
+4,4,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -8622,8 +8622,8 @@ ai.control=FALSE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
-4,5,0,16,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
+4,4,0,16,0,20,TRUE,FALSE
 5,5,0,0,0,0,FALSE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -10455,7 +10455,7 @@ is_alive=TRUE
 ai.control=FALSE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -10468,7 +10468,7 @@ diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_c
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
-3,5,0,0,0,20,FALSE,FALSE
+3,3,0,0,0,20,FALSE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -24867,11 +24867,11 @@ is_alive=TRUE
 ai.control=FALSE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
-4,5,0,0,0,20,TRUE,FALSE
+4,4,0,0,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
-3,5,0,0,0,20,FALSE,FALSE
+3,3,0,0,0,20,FALSE,FALSE
 5,5,0,0,0,0,FALSE,FALSE
 1,5,0,0,0,20,FALSE,FALSE
 1,5,0,0,0,20,FALSE,FALSE
